lib/krikri/registry.rb

```diff
@@ -0,0 +1,45 @@
+module Krikri
+
+  ##
+  # Provide a registry of defined items that can be looked up by a token
```

Comment:
Thanks, I recall the efficiency argument but just wasn't sure about converting back and forth.
I'll change it to a symbol if you think that's best.